src/rollup/widgets/construct-widget-layout.js

```diff
@@ -187,12 +187,14 @@ export default function constructWidgetLayout (widget, config) {
 
   // NOTE Set the image
   if (config.image) {
+    var widgetImage = document.createElement('img');
+    widgetImage.src = config.image;
+    widgetImage.className = 'pf-widget-img';
     if (config.layout === 'button') {
```

Comment:
Should rework this logic. It looks like we can clean it up so that there isn't two `widgetContent.appendChild(widgetImage);`.

Reply:
Where are you seeing two of them?

Reply:
Ignore. I misread `widgetContent` and `widgetBody`